tensorflow/lite/interpreter.cc

```diff
@@ -246,6 +246,14 @@ TfLiteStatus Interpreter::SetExecutionPlan(const std::vector<int>& new_plan) {
 void Interpreter::UseNNAPI(bool enable) { primary_subgraph().UseNNAPI(enable); }
 
 void Interpreter::SetNumThreads(int num_threads) {
+  // num_threads should be greater than -1.
```

Comment:
Might put these comments to the place where this API is declared in interpreter.h?